Data-tranformations/fairgenomes/samples.yarrml.yaml

```diff
@@ -0,0 +1,22 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
+  OBI: http://purl.obolibrary.org/obo/OBI_
+  EFO: http://www.ebi.ac.uk/efo/EFO_ # I added this one
+  GSSO: http://purl.obolibrary.org/obo/GSSO_ # I added this one
+
+mappings:
+  fairgenomes:
+    sources:
+      - [samples.csv~csv] # I am going to name it like this samples.csv
+    s: ega.dataset:$(accession_id) # I am unsure for this part, I have not changed
```

Comment:
Using the prefixes from the previous comment this line would become:
`    s: ega.samples:$(accession_id)`

Reply:
What about the most complicated mappings e.g. `study_analysis_sample`?